Suite2p/suite2p_parameters.py

```diff
@@ -20,14 +20,30 @@ def __init__(self, danse_params: dict):
         self.paths: dict          = danse_params.get(defs.Parameters.Main.PATHS, {})
         self.params: dict         = danse_params.get(defs.Parameters.Main.PARAMETERS, {})
         self.preview_params: dict = danse_params.get(defs.Parameters.Main.PREVIEW, {})
-        self.time_length: int      = self.get_time_length()
+        self.time_length: int     = self.get_time_length()
         
-        self.ops = suite2p.default_ops()
-        self.ops['batch_size']        = 50 # Decrease the batch_size in case low RAM on computer
-        self.ops['threshold_scaling'] = self.params['Threshold'] # Threshold for ROIs detection
-        self.ops['tau']               = self.params['DecayTime'] # Timescale of gcamp to use for deconvolution
-        self.ops['nplanes']           = len(self.paths[defs.Parameters.Path.H5PATH])
+        self.ops = suite2p.default_ops() # https://suite2p.readthedocs.io/en/latest/settings.html#
+        # Suite2p Main Settings
         self.ops['data_path']         = [self.paths[defs.Parameters.Path.TMP_DIR]]
+        self.ops['nplanes']           = len(self.paths[defs.Parameters.Path.H5PATH])
+        self.ops['tau']               = self.params['BiosensorDecayTime'] # Timescale of GCaMP to use for deconvolution
+
+        # Suite2p Registration Settings
+        self.ops['batch_size']        = 500 # Decrease the batch_size in case low RAM on computer
+        self.ops['smooth_sigma']      = 4   # STD in pixels of the gaussian used to smooth the phase correlation between the reference image and the frame which is being registered
+    
+        # Suite2p 1P registration
```

Comment:
Suite2p is designed to 2-photon microscope recordings, both voluscan and our microscope are 1-photon, so we kinda have to enable these parameters. Not sure about the values though, we should probably do an estimation based on the neuron diameter and image size. @4D42 @kapilseh what do you think?

Reply:
Yes true.